packages/batch-poster-monitor/index.ts

```diff
@@ -231,16 +268,6 @@ const monitorBatchPoster = async (childChainInformation: ChainInfo) => {
     transport: http(childChainInformation.orbitRpcUrl),
   })
 
-  // First, a basic check to get batch poster balance
```

Comment:
Moved this code block later in the code, where we also have `sequencerInboxLogs` generated to go in it as an input.